ci-operator/config/netobserv/network-observability-console-plugin/netobserv-network-observability-console-plugin-main.yaml

```diff
@@ -0,0 +1,143 @@
+base_images:
+  go-builder-base:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+  ubi:
+    name: ubi
+    namespace: ocp
+    tag: "9"
+  web-builder-base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "9"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+  use_build_cache: true
+images:
+- dockerfile_path: Dockerfile
+  from: ubi
+  inputs:
+    go-builder-base:
+      as:
+      - docker.io/library/golang:1.21
+    web-builder-base:
+      as:
+      - docker.io/library/node:16-alpine
+  to: network-observability-console-plugin
+- build_args:
+  - name: BUILDSCRIPT
+    value: :standalone
+  dockerfile_literal: |
```

Comment:
You'd also consider adding this dockerfile in the repo, as for the other image. Eventually, you could get into some changes for which you need to change both and you'll get tests failing for a kind of circular dependency between the involved PRs.